packages/create-expo-nightly/patches/packages/RCTPackagerConnection.patch

```diff
@@ -0,0 +1,57 @@
+diff --git a/packages/expo-dev-launcher/ios/EXDevLauncherController.m b/packages/expo-dev-launcher/ios/EXDevLauncherController.m
+index 97dc4923a7..c8f86fb220 100644
+--- a/packages/expo-dev-launcher/ios/EXDevLauncherController.m
++++ b/packages/expo-dev-launcher/ios/EXDevLauncherController.m
+@@ -444,9 +444,9 @@
+ 
+ #if RCT_DEV
+     // Connect to the websocket, ignore downloaded update bundles
+-    if (![bundleUrl.scheme isEqualToString:@"file"]) {
+-      [[RCTPackagerConnection sharedPackagerConnection] setSocketConnectionURL:bundleUrl];
+-    }
++//    if (![bundleUrl.scheme isEqualToString:@"file"]) {
++//      [[RCTPackagerConnection sharedPackagerConnection] setSocketConnectionURL:bundleUrl];
++//    }
```

Comment:
i haven't tested whether the commenting code is correct fix. when we upgrade our repo to react-native 0.84+, we should double check that again.

Reply:
adding many reviews to have a note for the uncertain change. i guess we don't determine who to upgrade react-native for sdk 56 yet.